assets/sass/components/surveys/_googlesitekit-survey-completion.scss

```diff
@@ -0,0 +1,34 @@
+/**
+ * User Feedback Completion styles.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.googlesitekit-survey__completion {
+
+	.googlesitekit-survey__body {
+		color: $c-scarpa-flow;
+		font-family: $f-primary;
+		font-size: 0.875rem;
+		font-weight: $fw-primary-normal;
+		line-height: 1.28;
+		padding: 0 $grid-gap-phone $grid-gap-phone 56px;
+
+		@media (min-width: $bp-desktop) {
+			padding-left: 70px;
```

Comment:
Many of the values I've seen (in other files too) are very specific. I'm guessing this is from the design, but it would be great if we could annotate any of these as to why they are that value (since we don't have a descriptive variable for it).

That or – if any of them are micro–styles to make it 100% pixel-perfect, maybe we don't need them. Everything looks great, I'm just a bit wary of adding too many special-case values as it can lead to inconsistency across the product.

Reply:
Updated. So actually I wouldn't say those are micro-styles, rather it's how the design is. For e.g in the `SurveyCompletion` component, the main content starts where the logo ends, which seems to me like a design decision. But I got your point :)